### PR TITLE
More details about MCAS integration token

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32982
fixes: https://github.com/demisto/etc/issues/32822

## Description
More details about the MCAS integration token and the kind of permission are needed for the token owner in Microsoft Cloud App Security.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 This is the MicrosoftCloudAppSecurity integration.
 This integration was integrated and tested with version 178 of MicrosoftCloudAppSecurity
+
+For more details about how to generate a new token, see [Microsoft Cloud App Security - Managing API tokens](https://docs.microsoft.com/en-us/cloud-app-security/api-authentication).
+
+For more details about the kind of permission are needed for the token owner in Microsoft Cloud App Security, see [Microsoft Cloud App Security - Manage admin access](https://docs.microsoft.com/en-us/cloud-app-security/manage-admins).
```
Comment (kirbles19) [inline Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md:6]: For more information about which permissions are required for the token owner in Microsoft Cloud App Security, see ....

File: Packs/MicrosoftCloudAppSecurity/ReleaseNotes/1_0_14.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Microsoft Cloud App Security
+- Improving documentation - added example for the **Custom Filter** parameter, and explanation about the the type and permissions assigned to the user/token used for the integration.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```
Comment (kirbles19) [inline Packs/MicrosoftCloudAppSecurity/ReleaseNotes/1_0_14.md:5]: - Updated the Docker image to: *demisto/python3:3.9.1.15759*.

**Review by kirbles19 — APPROVED** [review]
Done.

**Comment by kirbles19** [comment]
@ChanochShayner Done.

File: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md (hunk 1 above)
Comment (ChanochShayner) [reply Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md]: done

File: Packs/MicrosoftCloudAppSecurity/ReleaseNotes/1_0_14.md (hunk 2 above)
Comment (ChanochShayner) [reply Packs/MicrosoftCloudAppSecurity/ReleaseNotes/1_0_14.md]: done

**Pull request merged**